docs/guides/backup_security.md

```diff
@@ -33,17 +34,17 @@ You can use a copy of the `stronghold.snapshot` file as a backup. You can implem
 
 ## How to Restore From a Backup
 
-Simply place a snapshot file in the directory that wallet.rs expects.
+Place a snapshot file in the directory that wallet.rs expects.
 
 ## How to Export a User's Stronghold
 
 You can create a new Stronghold snapshot on the fly to allow a user to leave your service and retain their key.
 
 ## How to Rekey a Stronghold/Password Rotation
 
-The procedure for changing a Stronghold password is "simple": you read a snapshot into a vault and then write it out with a new encryption password. [See this code for the source.](https://github.com/iotaledger/wallet.rs/blob/d1b8893d73aae35dfcf7c5c8006e2177988d25d0/src/stronghold.rs#L436-L451)
+The procedure for changing a Stronghold password is simple: you read a snapshot into a vault and then write it out with a new encryption password. [See this code for the source.](https://github.com/iotaledger/wallet.rs/blob/d1b8893d73aae35dfcf7c5c8006e2177988d25d0/src/stronghold.rs#L436-L451)
```

Comment:
"The procedure for changing" can be a verb, it may or not be simple since it is a subjective, and I think you can make the sentence smoother without a colon.

I think it is enough to put only "code" into a link instead of the full sentence.

Reply:
Agree with all points except the last one; reading the word "code" without five words of context is a little jarring.

Reply:
I've meant "See this [code](https://github.com/iotaledger/wallet.rs/blob/d1b8893d73aae35dfcf7c5c8006e2177988d25d0/src/stronghold.rs#L436-L451) for the source."

Reply:
Perfect 👌